content/soarian/r4/clinical/diagnostics/diagnostic-report.md

```diff
@@ -0,0 +1,145 @@
+	* [Effective period](https://hl7.org/fhir/r4/diagnosticreport-definitions.html#DiagnosticReport.effective_period){:target="_blank"}
+* [Issued](https://hl7.org/fhir/r4/diagnosticreport-definitions.html#DiagnosticReport.issued){:target="_blank"}
+* [Performer](https://hl7.org/fhir/r4/diagnosticreport-definitions.html#DiagnosticReport.performer){:target="_blank"}
+* [Result](https://hl7.org/fhir/r4/diagnosticreport-definitions.html#DiagnosticReport.result){:target="_blank"}
+* [Presented form](https://hl7.org/fhir/r4/diagnosticreport-definitions.html#DiagnosticReport.presentedForm){:target="_blank"}
+
+
+## Terminology Bindings
+
+<%= terminology_table(:soarian_diagnosticreport, :r4) %>
+
+## Search
+
+Search for DiagnosticReport resources that meet the specified query parameters:
+
+	GET /DiagnosticReport?:parameters
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true)%>
+
+### Parameters
+
+ Name         | Required?          | Type          | Description
+--------------|--------------------|---------------|--------------
+ `_id`        | This, or `patient` | [`token`]     | The logical resource ID associated with the resource.
+ `patient`    | This, or `_id`     | [`reference`] | The patient that the diagnostic report is about. 
+ `category`   | No                 | [`token`]     | The category of the diagnostic report.
+ `code`       | No                 | [`token`]     | The code of the diagnostic report. 
+ `encounter`  | No                 | [`reference`] | The encounter associated with the care plan record. 
+ `date`       | No                 | [`date`]      | Date range into which the diagnostic report falls. Example: `date=gt2014-09-24 or date=lt2015-09-24T12:00:00.000Z`
+ `_count`     | No                 | [`count`]     | The maximum number of resources returned in a page.
+ `_revincude` | No                 | [`_revinclude`]| A request to include any Provenance resource in the bundle that refers to a DiagnosticReport resource in the search results. Only supported with Provenance.
+
+ Notes
+
+*	The `date` parameter may be provided up to two times, and must use the `eq`, `ge`, `gt`, `le`, or `lt` prefixes. When a value is provided without a prefix, an implied `eq` prefix is used. When provided twice, the lower value must have a `ge` or `gt` prefix and the higher value must have an `le` or `lt` prefix.
+* The `_revinclude` parameter may be provided once with the value `Provenance:target`. Example: `_revinclude=Provenance:target`
+* When `_revinclude` is provided in a request, the OAuth2 token must include the `patient/Provenance.read  system/Provenance.read`  or  `user/Provenance.read` scope as applicable.
+
+
+### Headers
+
+<%= headers fhir_json: true %>
+
+### Example Search by Patient
+
+#### Request
+
+	GET https://fhir-myrecord-sc.sandboxcerner.com/r4/3f2aca24-87f3-4eac-a6d7-1f75247e6b43/DiagnosticReport?patient=A879904FD2FE4B2D90C89FDA84E1285F
```

Comment:
We should probably be giving the public sandbox URL I suppose, could you please cross check and implement in all other instances as well

Reply:
You can update the public sandbox URL once implemented, will approve the PR